code/vmix.c

```diff
@@ -47,6 +47,7 @@
 #include "vm.h"
 
```

Comment:
General comment: Some comments from protix.c also apply to corresponding places in this file